shopping_mall/backend/tests/conftest.py

```
@@ -0,0 +1,242 @@
+    chat_session_mock.filter.return_value.first.return_value = chat_session
+
+    db.add = MagicMock()
+    db.commit = MagicMock()
+    db.refresh = MagicMock()
+    return db
+
+
+# ── 도메인 오브젝트 팩토리 ───────────────────────────────────────────────────
+
+def make_order(
+    id=1,
```

Ruff (0.15.9)

[error] Function argument `id` is shadowing a Python builtin

(A002)